libs/sql-schema-describer/tests/describer_tests.rs

```diff
@@ -15,43 +16,43 @@ mod postgres;
 mod sqlite;
 mod test_api;
 
-fn int_full_data_type(api: &TestApi) -> String {
-    match (api.sql_family(), api.connector_name()) {
-        (SqlFamily::Postgres, _) => "int4".to_string(),
-        (SqlFamily::Sqlite, _) => "INTEGER".to_string(),
-        (SqlFamily::Mysql, "mysql8") => "int".to_string(),
-        (SqlFamily::Mysql, _) => "int(11)".to_string(),
-        (SqlFamily::Mssql, _) => "int".to_string(),
+fn int_full_data_type(api: &TestApi) -> &'static str {
+    match api.sql_family() {
```

Comment:
could this completely operate on tags?

Reply:
yes, but I think it's a bit ugly (it's basically `if tags.contains(Mysql) { ... } else if ... { ... }`). I'm sure we'll find a way to make it pretty.